libbeat/cfgfile/reload.go

```diff
@@ -137,7 +140,8 @@ func (rl *Reloader) Check(runnerFactory RunnerFactory) error {
 		if !c.Config.Enabled() {
 			continue
 		}
-		_, err := runnerFactory.Create(rl.pipeline, c.Config, c.Meta)
+		instance, err := runnerFactory.Create(rl.pipeline, c.Config, c.Meta)
+		instance.Stop()
```

Comment:
Do you know what the affect here on other Beats are? I'm still concerned we might break something with it.